server/api.js

```diff
@@ -1,31 +1,46 @@
-"use strict";
+"use strict"
 
-module.exports = function(db) {
-	var handle = new apiHandler(db)
-	return (function(handle) {
-		return function() {
-			handle.handle.apply(handle, arguments)
-		}
-	})(handle)
+const fail = function(res, msg) {
+	res.status(503)
+	res.send(msg)
+}
+
+const prep = function(res) {
+	return function(data) {
+		res.type("application/json")
+		res.send(data)
+	}
 }
 
-var apiHandler = function(db) {
+const denodeify = function(fn, args, alt, th) {
+	return new Promise(function(resolve, reject) {
+		args[args.length] = (function(err, data) {
+			if (err) {
+				reject(err || "No Data")
+			} else {
+				resolve(alt ? alt(data) : data)
+			}
+		})
+		fn.apply(th, args)
+	})
+}
+
+const apiHandler = function(db) {
 	this.db = db
 }
 
 apiHandler.prototype = {
-	handle:
-	function(req, res, next) {
-		var match = req.originalUrl.match(/\/api\/([a-z0-9]+)\/([a-z0-9]+)/)
+	handle: function(req, res, next) {
+		const match = req.originalUrl.match(/\/api\/([a-z0-9]+)\/([a-z0-9]+)/)
 		if (match) {
-			var out
-			if (match[1] == "get") {
+			let out
```

Comment:
I've refactored API to a class and these if/else to an object, should I include it here (even though it's not directly related with ESLint)?